LeetCodeSolutions/Program.cs

```diff
@@ -1,3 +1,30 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿namespace LeetCodeSolutions;
+public class Program
+{
+    public int[] TwoSum(int[] nums, int target)
+    {
+        int[] outcome = new int[2];
+        var passedElements = new Dictionary<int, int>();
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+        for(var i = 0;i < nums.Length;i++)
+        {
+            var neededNumber = target - nums[i];
+            if (passedElements.ContainsKey(neededNumber))
+            {
```

Comment:
based on the questions constraint, there is only one combination in the list that yields this sum, if we found, no need to continue looping through the other elements

Reply:
Logically, I agree with this. But when I added a break keyword to my solution (when finding the needed number) the program actually took more time based on the run time submission (246ms without break vs 300ms with break) from LeetCode. I've even ran it multiple times just to make sure. So I'll look more into it and see if I can provide a logical explanation of the cause.